src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/AdminDashboardPage.java

```diff
@@ -2,7 +2,14 @@
 
 import org.apache.wicket.authroles.authorization.strategies.role.Roles;
 import org.apache.wicket.authroles.authorization.strategies.role.annotations.AuthorizeInstantiation;
+import org.apache.wicket.markup.html.basic.Label;
+import pl.tomaszdziurko.jvm_bloggers.view.admin.counters.NewPostsCounterModel;
 
 @AuthorizeInstantiation(Roles.ADMIN)
 public class AdminDashboardPage extends AbstractAdminPage {
+
+    public AdminDashboardPage() {
+        Label postsSinceLastPublicationLabel = new Label("postsSinceLastPublication", new NewPostsCounterModel());
```

Comment:
Unnecessary one-time variable. I would rather do this like that without loosing readability:
```` java
 add(new Label("postsSinceLastPublication", new NewPostsCounterModel()));
````